packages/bootstrap/README.md

```diff
@@ -0,0 +1,20 @@
+# @kaizen/bootstrap
```

Comment:
Naming thoughts

* Clash with another famous "bootstrap"
* `KaizenContainer`?
* `Kontainer`?
* `KaizenContext`?
    * We could use React Context for interesting things, like a single resize observer at the root of the page
    * And a ThemeProvider?
* `KaizenProvider`?

Reply:
I've changed to `Container` (for now). Naming things is hard.